prow/gcpproxy-coverage.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Presubmit script triggered by Prow.
+
+# Fail on any error.
+set -eo pipefail
+
+gcloud config set core/project cloudesf-testing
+gcloud auth activate-service-account \
```

Comment:
Maybe login by referencing env instead of absolute path?
BTW, I looked at our other jobs, which all include the project setting step but no activate-service-account step t so maybe you can try removing no activate-service-account step

Reply:
Sure, let's use env var.

I double checked some other jobs, it seems like they aren't using the correct service account. They are still using the default prow one as far as I can tell. Take a look at the first few lines here: https://oss-prow.knative.dev/view/gcs/oss-prow/pr-logs/pull/GoogleCloudPlatform_esp-v2/56/ESPv2-presubmit/1237771179012395010

Let's keep the activation in this script. I'm make another PR to either make this consistent across all non-kubekins scripts or try to debug why it's not activated by default.